.env.example

```
@@ -133,22 +135,41 @@ WATSONX_PROJECT_ID=
 LLM_PROVIDER=
 LLM_MODEL=
 
+# OPTIONAL: Custom system prompt for the chat agent. Overrides the default prompt baked into the Langflow flow.
+# SYSTEM_PROMPT=
+
 # Embedding provider configuration. Providers can be "watsonx", "ibm" or "ollama".
 EMBEDDING_PROVIDER=
 EMBEDDING_MODEL=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The EMBEDDING_MODEL key should go before the EMBEDDING_PROVIDER key

(UnorderedKey)